ReSharper.FSharp/test/data/features/daemon/identifierHighlighting/Delegates 02.fs.gold

```diff
@@ -0,0 +1,29 @@
+﻿open |System|(0)
+
+type |T1|(1) = |int|(2) -> |int|(3)
+
+let |f1|(4) (|f|(5) : |T1|(6)) = |f|(7) 1
+let |f2|(8) (|f|(9) : |Func|(10)<|int|(11), |int|(12)>) = |f|(13).|Invoke|(14)(1)
+let |f3|(15) (|f|(16) : |int|(17) -> |int|(18)) = |f|(19) 1
+
+---------------------------------------------------------
+(0): ReSharper F# Namespace Identifier: 
+(1): ReSharper F# Delegate Identifier: 
+(2): ReSharper F# Struct Identifier: 
+(3): ReSharper F# Struct Identifier: 
+(4): ReSharper F# Function Identifier: 
+(5): ReSharper F# Function Identifier: 
+(6): ReSharper F# Delegate Identifier: 
+(7): ReSharper F# Function Identifier: 
+(8): ReSharper F# Function Identifier: 
+(9): ReSharper F# Value Identifier: 
```

Comment:
It looks a bit inconsistent to `f` in `f1` and `f3`, though, I'm not sure what we want here. Maybe it's OK to check if the value type is a delegate entity.

Reply:
Yeah. That's a tricky one. The idea was to show that a "true delegate" parameter like func or action or delegate of... is a value. Because highlighting it as function is a bit wierd.. People might think it can be called like f(x) or f x, which is wrong. I can remove it if you want.

Reply:
@auduchinok Eugene, can you please tell me whether I should delete this test case or leave it?

Reply:
> People might think it can be called like f(x) or f x or x |> f, which is wrong. I can remove it if you want.

Yeah, it totally makes sense indeed. Let's leave it as is.